controller/userController.js

```diff
@@ -0,0 +1,23 @@
+const { auth, firestore } = require("firebase-admin")
+const { addUserProfilePhoto } = require("../helper/helperFunction")
+const userRegister = async (req, res) => {
+  const { firstName, lastName, mobileNo, email, password, profilePhoto, confirmPassword } = req.body;
+  if (!firstName || !lastName || !mobileNo || !email || !password || !confirmPassword || !profilePhoto) {
```

Comment:
can you use Exapress validator to validate these fields @ashok-choudhary-0 https://express-validator.github.io/docs/